src/aat/resources/features/F-107/S-928.td.json

```diff
@@ -25,11 +25,11 @@
     }
   },
   "expectedResponse": {
-    "_extends_": "Common_400_Response",
+    "_extends_": "Common_422_Response",
     "body" : {
-      "exception" : "uk.gov.hmcts.ccd.endpoint.exceptions.BadRequestException",
+      "exception" : "uk.gov.hmcts.ccd.endpoint.exceptions.CaseValidationException",
```

Comment:
Now this is returning a normal `CaseValidationException`, we should also see the specific errors in the `details` field further down - no longer `null`.

Reply:
sorry for the late response, Yes I agreed it was implemented locally and pushed. Thanks for flag it .